sdk/communication/communication-identity/test/public/communicationIdentityClient.spec.ts

```diff
@@ -6,61 +6,123 @@ import {
-  });
+    it("successfully creates a user", async function() {
+      const user: CommunicationUserIdentifier = await client.createUser();
+      assert.isString(user.communicationUserId);
+    });
 
-  it("successfully creates a user and token", async function() {
-    const { user: newUser, token } = await client.createUserAndToken(["voip"]);
-    assert.isString(newUser.communicationUserId);
-    assert.isString(token);
-  });
+    it("successfully creates a user and token", async function() {
+      const { user: newUser, token } = await client.createUserAndToken(["voip"]);
+      assert.isString(newUser.communicationUserId);
+      assert.isString(token);
+    });
 
-  it("successfully gets a token for a user [single scope]", async function() {
-    const { token, expiresOn } = await client.getToken(user, ["chat"]);
-    assert.isString(token);
-    assert.instanceOf(expiresOn, Date);
-  });
+    it("successfully gets a token for a user [single scope]", async function() {
+      const user: CommunicationUserIdentifier = await client.createUser();
+      const { token, expiresOn } = await client.getToken(user, ["chat"]);
+      assert.isString(token);
+      assert.instanceOf(expiresOn, Date);
+    });
 
-  it("successfully gets a token for a user [multiple scopes]", async function() {
-    const { token, expiresOn } = await client.getToken(user, ["chat", "voip"]);
-    assert.isString(token);
-    assert.instanceOf(expiresOn, Date);
-  });
+    it("successfully gets a token for a user [multiple scopes]", async function() {
+      const user: CommunicationUserIdentifier = await client.createUser();
+      const { token, expiresOn } = await client.getToken(user, ["chat", "voip"]);
+      assert.isString(token);
+      assert.instanceOf(expiresOn, Date);
+    });
 
-  it("successfully creates a user and gets a token in a single request", async function() {
-    const { user: newUser, token, expiresOn } = await client.createUserAndToken(["chat", "voip"]);
-    assert.isTrue(isCommunicationUserIdentifier(newUser));
-    assert.isString(token);
-    assert.instanceOf(expiresOn, Date);
-  });
+    it("successfully creates a user and gets a token in a single request", async function() {
+      const { user: newUser, token, expiresOn } = await client.createUserAndToken(["chat", "voip"]);
+      assert.isTrue(isCommunicationUserIdentifier(newUser));
+      assert.isString(token);
+      assert.instanceOf(expiresOn, Date);
+    });
 
-  it("successfully revokes tokens issued for a user", async function() {
-    await client.revokeTokens(user);
-  });
+    it("successfully revokes tokens issued for a user", async function() {
+      const { user } = await client.createUserAndToken(["chat", "voip"]);
+      await client.revokeTokens(user);
+    });
+
+    it("successfully deletes a user", async function() {
+      const user: CommunicationUserIdentifier = await client.createUser();
+      await client.deleteUser(user);
+    }).timeout(20000);
```

Comment:
Does the timeout need to be this long?

Reply:
I actually don't know the reasoning behind this, but it probably doesn't. I'm curious to know why this was decided in the first place.

Reply:
@DominikMe git tells me that this timeout was set by you :) can this be lower?

Reply:
@DominikMe I'm all for lower timeouts, let's try it. I must have set it higher because it was timing out I guess?